test/quantization/core/test_quantized_tensor.py

```diff
@@ -903,3 +903,65 @@ def test_choose_qparams_optimized(self):
             ref = param_search_greedy(x.numpy(), bit_rate=bit_width)
             self.assertEqual(y[0].numpy(), ref[0])
             self.assertEqual(y[1].numpy(), ref[1])
+
+class TestQTensorSerialization(TestCase):
```

Comment:
shall we just merge this with TestQuantizedTensor?

Reply:
sure